.github/workflows/server-ci-report.yml

```
@@ -86,12 +86,13 @@ jobs:
         env:
           TEST_NAME: "${{ matrix.test.name }}"
           FLAKY_SUMMARY: "${{ steps.report.outputs.flaky_summary }}"
+          PR_NUMBER: "${{ steps.incoming-pr.outputs.NUMBER }}"
         with:
           script: |
             const body = `#### ⚠️  One or more flaky tests detected ⚠️\n* Failing job: [github.com/mattermost/mattermost:${process.env.TEST_NAME}](${{ github.event.workflow_run.html_url }})\n* Double check your code to ensure you haven't introduced a flaky test.\n* If this seems to be unrelated to your changes, submit a separate pull request to skip the flaky tests (e.g. [23360](https://github.com/mattermost/mattermost/pull/23360)) and file JIRA ticket (e.g. [MM-52743](https://mattermost.atlassian.net/browse/MM-52743)) for later investigation.\n\n${process.env.FLAKY_SUMMARY}`
 
             await github.rest.issues.createComment({
-              issue_number: ${{ steps.incoming-pr.outputs.NUMBER }},
+              issue_number: ${process.env.PR_NUMBER},
```

Shouldn't it be like this? We're not in a template literal here. 
```suggestion
              issue_number: process.env.PR_NUMBER,
```